src/api/json/catalog.json

```diff
@@ -5420,7 +5420,7 @@
       "name": "Windows Package Manager Locale Manifest",
       "description": "Windows Package Manager Locale Manifest file, used for detailing locale specific metadata",
       "fileMatch": [
-        "**/manifests/?/*/*/*/*.*.locale@(.en-US|fr-FR|it-IT|ja-JP|ko-KR|pt-BR|ru-RU|zh-CN|zh-TW).yaml"
+        "**/manifests/?/*/*/*/*.*.locale.{en-US,fr-FR,it-IT,ja-JP,ko-KR,pt-BR,ru-RU,zh-CN,zh-TW}.yaml"
```

Comment:
If I understand correctly this used to match `x.y.locale.en-EN.yaml` and `x.y.localefr-FR.yaml`. Now it matches `x.y.locale.en-EN.yaml` and `x.y.locale.fr-FR.yaml`. (Notice the extra dot for non-English locales.)

Reply:
Yeah the original pattern seemed incorrect